website/docs/components/badge/partials/guidelines/guidelines.md

```diff
@@ -124,8 +124,16 @@ Badges come in a few icon and text combinations; text only, icon only, and icon
 <Hds::Badge @color="neutral" @type="filled" @icon="corner-down-left" @isIconOnly={{true}} @text="Icon only" />
 <Hds::Badge @color="neutral" @type="filled" @icon="hexagon" @text="Text + Icon" />
 
+### Usage in status Badges
+
+To avoid relying on color alone, we recommend including an icon in Badges that communicate status (Success, Warning, Critical).
```

Comment:
We opted to make this change because there are other ways to avoid relying on color, such as using specific language in the badge ("Success", "Error", etc). Right now it comes across as a badge would be relying on color alone if an icon isn't included. Is there a way to provide more clarity?